tests/e2e/specFiles/Transactions/send.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { name } from 'dayjs/locale/*';
+import { expect } from 'detox';
+
+import { isBeta, createRandomNote, WALLETS_WITH_COINS, DATA_FOR_TRANSACTIONS } from '../../helpers';
+import app from '../../pageObjects';
+
+describe('Transactions', () => {
+  beforeEach(async () => {
+    isBeta() && (await app.onboarding.betaVersionScreen.close());
+    await app.developerRoom.tapOnSkipOnboardingButton();
+    await app.navigationBar.changeTab('wallets');
+  });
+
+  describe('Send', () => {
+    describe('@android @ios @regression', () => {
```

Comment:
I think that making transaction is so crucial feature in our app that at least one test should find a place in `@smoke` suite